debian/control

```diff
@@ -94,6 +94,14 @@ Package: eos-payg-data
 Section: misc
 Architecture: all
 Depends:
+ eos-paygd (= ${binary:Version}),
```

Comment:
I would consider making this a separate `eos-payg-tools` package, but I don’t mind either way.

Reply:
I did consider this but I decided that adding 1 more package and then go list it in eos-meta isn't a net win.

Agreed that a manpage would be nice but it's a marginal win over improving the `--help` output (at which point one might generate it at package build time with help2man).